src/components/organisms/DropdownContent/DropdownContent.tsx

```diff
@@ -0,0 +1,151 @@
+import { Dispatch, FC, SetStateAction, useState } from 'react'
+import classNames from 'classnames'
+import { includes, map } from 'lodash'
+import CheckBoxInput from 'components/molecules/CheckBoxInput/CheckBoxInput'
+import Button, {
+  AppearanceTypes,
+  SizeTypes,
+} from 'components/molecules/Button/Button'
+import { SelectionControlsInputProps } from 'components/organisms/SelectionControlsInput/SelectionControlsInput'
+
+import classes from './styles.module.scss'
+
+type DropdownContentProps = SelectionControlsInputProps & {
+  isOpen?: boolean
+  selectedOptionObjects?: { label: string; value: string }[]
+  setIsOpen?: Dispatch<SetStateAction<boolean>>
+}
+
+const DropdownContent: FC<DropdownContentProps> = ({
+  dropdownSize = 'l',
+  disabled,
+  isOpen,
+  searchInput,
+  options,
+  multiple = false,
+  value,
+  name,
+  ariaLabel,
+  buttons = false,
+  cancelButtonLabel,
+  proceedButtonLabel,
+  onChange,
+  setIsOpen,
+  helperText,
+  selectedOptionObjects,
+  tags,
+}) => {
+  const [selectedValue, setSelectedValue] = useState<string[]>(
+    value ? (Array.isArray(value) ? value : [value]) : []
+  )
+
+  const handleSingleSelect = (selectedOption: string) => {
+    onChange(selectedOption ? selectedOption : '')
+  }
+
+  const handleMultipleSelect = (selectedOption: string) => {
+    const selectedValues = Array.isArray(selectedValue)
+      ? [...selectedValue]
+      : []
+    const optionIndex = selectedValues.indexOf(selectedOption)
+
+    const newSelectedValues =
+      optionIndex === -1
+        ? [...selectedValues, selectedOption]
+        : selectedValues.filter((value) => value !== selectedOption)
+
+    setSelectedValue(newSelectedValues)
+
+    if (setIsOpen) {
+      setIsOpen(true)
+    }
+  }
+
+  const handleOnSave = () => {
+    onChange(selectedValue)
+  }
+
+  const handleCancel = () => {
```

Comment:
**[blocker]**

Currently this does not seem to work correctly. You are clearing the internal state of DropdownContent only.

1. You are changing the defaultvalues passed to an empty array, which means "Tühista" doesn't actually reset the changes the user made, but rather acts as "clear all".

2. Since you are only calling this on the internal state of DropdownContent, then the value in the form will still remain correct. Currently this means that we are displaying different information inside the DropdownContent and under the input, where we show selected values